apps/price_pusher/src/evm/evm.ts

```diff
@@ -329,11 +448,38 @@ export class EvmPricePusher implements IPricePusher {
 }
 
 export class PythContractFactory {
```

Comment:
Is this export actually consumed?  I know this wasn't introduced in this code but IMO this pattern significantly overcomplicates things.  It adds a significant amount of abstraction and indirection for what is fundamentally just some constants for default `getContract` config or perhaps a function or two wrapping it and adding default config.

I don't think this is an appropriate use of `class`.  It isn't stateful and really has no good benefit to being encapsulated in this way.  I'd suggest considering replacing this (perhaps in a follow up) with some simple types / objects and functions and reducing the amount of abstractions.